pkg/core/config/message.go

```diff
@@ -0,0 +1,64 @@
+package config
+
+//go:generate msgp -tests=false
+
+type MessagesConfiguration struct {
+	Email       EmailMessageConfiguration `json:"email,omitempty" yaml:"email" msg:"email"`
+	SMSProvider SMSProvider               `json:"sms_provider,omitempty" yaml:"sms_provider,omitempty" msg:"sms_provider"`
+	SMS         SMSMessageConfiguration   `json:"sms,omitempty" yaml:"sms" msg:"sms"`
```

Comment:
It is a bit strange we have both `sms_provider` and `sms` at the same level. I understand we need them separate because SMSMessageConfiguration could appear in feature configuration.

Reply:
I do think it looks weird... but I can't think of a better name. I had an idea of moving the message configuration one level deeper, so it would be `messages.email.message`, `messages.sms.message`, and `messages.sms.provider`, but it seems even weirder.

Any suggestions?

Reply:
No good idea :(

Just keep this.